src/extensions/default/RelatedFiles/main.js

```diff
@@ -0,0 +1,202 @@
+                    }
+                    else if (PathUtils.isAbsoluteUrl(linkHref)) {
+                        searchResult = getLinkFromLocalURL(linkHref);
+                    }
+                    else {
+                        fileURI = getAbsoultePath(currentDocument.file.fullPath, linkHref);
+                        searchResult = getLinkFromLocalURL(fileURI);
+                    }
+                    if (searchResult) {
+                        relatedFiles.push(searchResult);    
+                    }
+                }
+            }    
+            // Search for src in script tag to get related files.
+            for (i = 0; i < scriptTags.length; i++) {
+                var scriptSrc = scriptTags[i].getAttribute("src");
+                if (scriptSrc !== null) {
+                    if (isRemoteURL(scriptSrc)) {
+                        searchResult = getLinkFromRemoteURL(scriptSrc);
+                    }
+                    else if (PathUtils.isAbsoluteUrl(scriptSrc)) {
+                        searchResult = getLinkFromLocalURL(scriptSrc);
+                    }
+                    else {
+                        fileURI = getAbsoultePath(currentDocument.file.fullPath, scriptSrc);
+                        searchResult = getLinkFromLocalURL(fileURI);
+                    }
+                    if (searchResult) {
+                        relatedFiles.push(searchResult);    
+                    }
+                }
+            }
+        }
+    }
+
+    /**
+    * Check whether a given string is remote url.
+    * @param {string} tagurl The url which is to be checked for remote link.
+    * @returns {Boolean} true if the url matches regular expression for remote url.
+    **/
+	function isRemoteURL(tagurl) {
+		var pattern = RegExp('^((http|https|ftp)?:?\/\/)');
+        return pattern.test(tagurl); 
+	} 
+    
+    /**
+    * Create a SearchResult object which has path information about the local file.
+    * @params {string} tagurl The url from which file path information is to be extracted.
+    * @returns {Object} SearchResult object which has file path information.
+    **/
+    function getLinkFromLocalURL(tagurl) {
+        var searchResult = new StringMatch.SearchResult (tagurl);
+        var file = Filesystem.getFileForPath(tagurl);
+        searchResult.label = file._name;
+        searchResult.fullPath = file._path;
+        searchResult.stringRanges = [{
+            text: file._path,
+            matched: false,
+            includesLastSegment: true
+        }];
+        return searchResult;
+    }
+    
+    /**
+    * Create a SearchResult object which has path information about the remote file url.
+    * @params {string} tagurl The url from which file path information is to be extracted.
+    * @returns {Object} SearchResult object which has remote link path information.
+    **/
+    function getLinkFromRemoteURL (tagurl) {
+        var regex = /:\/\/(.[^/]+)/;
```

Comment:
Instead of using regex here, you can simply use `FileUtils` to get the file name which can be used as `label`.